pages/tutorials/3.0/window/inputs.md

```diff
@@ -69,13 +74,19 @@ sf::Mouse::setPosition(sf::Vector2i(10, 50));
 sf::Mouse::setPosition(sf::Vector2i(10, 50), window); // window is a sf::Window
 ```
 
-There is no function for reading the current state of the mouse wheel. Since the wheel can only be moved relatively, it has no absolute state that can be queried. By looking at a key you can tell whether it's pressed or released. By looking at the mouse cursor you can tell where it is located on the screen. However, looking at the mouse wheel doesn't tell you which "tick" it is on. You can only be notified when it moves (`MouseWheelScrolled` event).
+There is no function for reading the current state of the mouse wheel.
+Since the wheel can only be moved relatively, it has no absolute state that can be queried.
+By looking at a key you can tell whether it's pressed or released.
+By looking at the mouse cursor you can tell where it is located on the screen.
+However, looking at the mouse wheel doesn't tell you which "tick" it is on.
+You can only be notified when it moves (`MouseWheelScrolled` event).
```

Comment:
Might make sense to link to the event page

Reply:
`MouseWheelScrolled` the API docs or the events tutorial?

Reply:
Events tutorial